### PR TITLE
Touch up

### PR DESCRIPTION
**Pull request by ostreifel**
Additionally this repository should be renamed to vsts-picklist-colors-control and all of the github links updated.

### CODE REVIEW

File: details.md [hunk 1]
```diff
@@ -1,16 +1,14 @@
-> This is currently only available on TFS "15" or later. It is not available on VSTS.
```
Comment (cschleiden) [inline details.md:1]: I think we should include that it's now available on VSTS

File: details.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 
 You can customize the colors and labels you would like to appear on your control. Additionally, the control can be used by both your mouse and keyboard (space bar, left/right and up/down arrow keys).
  
-## Quick steps to get started ##
+## Quick steps to get started ## 
 
-> Please note: This extension is not yet supported on VSTS, as users cannot customize the control via VSTS UI. We will need to include steps on how to walk the user through this control for VSTS once there is support available. 
-
-1. Navigate to your work item form and choose any integer or string field you would like to replace with the toggle control.
+1. Navigate to your work item form custimization page and add a Picklist colors custom control.
```
Comment (cschleiden) [inline details.md]: spelling

Comment (cschleiden) [inline details.md]: picklist colors custom control?

File: vss-extension.json [hunk 3]
```diff
@@ -1,44 +1,41 @@
 {
     "manifestVersion": 1,
-    "id": "color-control",
+    "id": "picklist-colors",
     "version": "0.1.1",
-    "name": "Color Form Control",
+    "name": "Picklist Colors",
```
Comment (cschleiden) [inline vss-extension.json]: Granted, the old name wasn't great, but is this really better?

File: vss-extension.json [hunk 4]
```diff
@@ -1,44 +1,41 @@
...
     "description": "The example we built adds color to picklists and allows user customization through labels.",
-    "publisher": "<your-publisher>",
+    "publisher": "MS-Devlabs",
```
Comment (cschleiden) [inline vss-extension.json]: nit: lowercase

File: details.md (hunk 1 above)
Comment (ostreifel) [reply details.md:1]: Updated

File: details.md (hunk 2 above)
Comment (ostreifel) [reply details.md]: Reverted to old name

File: vss-extension.json (hunk 3 above)
Comment (ostreifel) [reply vss-extension.json]: Reverted

File: vss-extension.json (hunk 4 above)
Comment (ostreifel) [reply vss-extension.json]: Updated

File: details.md (hunk 2 above)
Comment (ostreifel) [reply details.md]: Updated

**Comment by obvioussean** [comment]
GitHub lets you rename a repository, you should probably do that.

**Review by obvioussean — APPROVED** [review]

**Pull request merged**